cmd/cmd.go

```diff
@@ -159,7 +159,49 @@ func RunHandler(cmd *cobra.Command, args []string) error {
 		return err
```

Comment:
This file has a lot of new code, but it should probably have less/similar than before if we're switching to use the `/api/chat` API for it